internal/database/migrations.go

```diff
@@ -1015,4 +1017,48 @@ var migrations = []func(tx *sql.Tx, driver string) error{
 		_, err = tx.Exec(sql)
 		return err
 	},
+	func(tx *sql.Tx, _ string) (err error) {
```

Comment:
similar to a previous `users` migration ([link](https://github.com/miniflux/v2/blob/bae08be7200f0f0c7398bd5a99a8dc79814f93bd/internal/database/migrations.go#L450)) this migration

1. Finds empty `external_id`'s
2. Updates each `external_id` to a random hex value
3. Commits

This could be done purely in SQL, but the alternatives are way more involved:

1. Pull in pgcrypto which would require pulling in OpenSSL as a dep and enabling the extension
2. Hand write a random function for a one-off ([example from brandur.org](https://brandur.org/fragments/secure-bytes-without-pgcrypto))